files/templates/mbta/login/resources/js/form-registration.js

```diff
@@ -0,0 +1,38 @@
+/* Registration Form Script */
+
+//window.onload = function() {
+//  setTimeout(
+//    function() {
+//      $("#token").val('this must be empty').delay(2000).queue(function() {
+//				$("#token").val((new Date).getFullYear());
+//				});
+//		
+//    }, 2000);
+//}
+
+
+(function() {
+
+	"use strict";
+
+	var webForm = {
+
+		initialized: false,
+
+		initialize: function() {
+			if (this.initialized) return;
+			this.initialized = true;
+
+			this.events();
+		},
+
+		events: function() {
+			$("#token").val('this must be empty').delay(2000).queue(function() {
+				$("#token").val((new Date).getFullYear());
```

Comment:
question: it looks like this is adding a `token` field to the form with the current year; can you say more about the purpose of the `token` field?

Reply:
This is a part of our simple anti-robot / anti-spam javascript code.